backend/sirius-web-spring-collaborative/src/main/java/org/eclipse/sirius/web/spring/collaborative/projects/EditingContextEventProcessor.java

```diff
@@ -188,6 +193,7 @@ private void publishEvent(IInput input, Optional<IPayload> optionalPayload) {
             EventHandlerResponse response = optionalResponse.get();
 
             this.disposeRepresentationIfNeeded();
+            this.representationService.deleteDanglingRepresentations(this.editingContext.getId());
```

Comment:
Now that we let the database detect which representations are dangling, we must call this *after* persisting the editing context so that the database sees the correct / up to date state.